charts/apisix/templates/configmap.yaml

```diff
@@ -166,7 +166,7 @@ data:
       error_log: "{{ .Values.logs.errorLog }}"
       error_log_level: "{{ .Values.logs.errorLogLevel }}"         # warn,error
       worker_processes: "{{ .Values.nginx.workerProcesses }}"
-      enable_cpu_affinity: {{ default "true" .Values.nginx.enableCPUAffinity }}
+      enable_cpu_affinity: {{ and true .Values.nginx.enableCPUAffinity }}
```

Comment:
Is the `and` for avoiding a missing `enableCPUAffinity`

Reply:
`default` tempalte function will make `enable_cpu_affinity` property always been `true`.